papermill/execute.py

```diff
@@ -69,7 +67,7 @@ def preprocess(self, nb, resources):
             cell.outputs = []
 
     # Execute each cell and update the output in real time.
-    with futures.ThreadPoolExecutor(max_workers=1) as executor:
+    with futures.ThreadPoolExecutor(max_workers=4) as executor:
```

Comment:
Why increase this? I thought the reason for using a thread pool here is to move the execution of the notebook to somewhere else, not to run things in parallel?

Reply:
@betatim Yeah, here I just want to try, I will change it back. :)